hub2labhook/__init__.py

```diff
@@ -1,5 +1,23 @@
 # -*- coding: utf-8 -*-
+import os
+import subprocess
 
 __author__ = 'Antoine Legrand'
 __email__ = '2t.antoine@gmail.com'
 __version__ = '0.3.2'
+
+
+def _get_git_sha():
+    if os.path.exists("GIT_HEAD"):
+        with open("GIT_HEAD", "r") as f:
```

Comment:
will be created in the dockerfile. 
How could it be generated from the setup.py when creating a pip release ?